syz-fuzzer/proc.go

```diff
@@ -123,6 +123,7 @@ func (proc *Proc) triageInput(item *WorkTriage) {
 	// Compute input coverage and non-flaky signal for minimization.
 	notexecuted := 0
 	rawCover := []uint32{}
+	proc.resetAccState()
```

Comment:
Triage is the most important step for corpus growth, don't we want to reset the state before each attempt? If the program itself creates accumulated state, then second and third runs won't give the coverage and we will discard it.
If we reset state during fuzzing and minimization, few additional resets here shouldn't make significant performance difference.

Reply:
Indeed, this could help avoid the state created by the program itself.

> If we reset state during fuzzing and minimization, few additional resets here shouldn't make significant performance difference.

It will make corpus triage significantly slower, but in normal operation yes, it should not be noticeable.

Reply:
We could persist coverage and reuse it if kernel haven't changed. Should eliminate more than half re-triages. And for some rarely changing trees, most of the re-triages.

Reply:
That's an interesting option, though I think it'd be somewhat out of scope of the experiment. 

Also, since we plan to run it on a -next tree, the chances are that we'd run it on a different kernel image every day anyway.